cloud-sql/postgres/activerecord/README.md

```diff
@@ -80,19 +80,15 @@ To use a Unix socket, you'll need to create a directory and give write access to
 the proxy. For example:
 
 ```bash
-sudo mkdir /cloudsql
-sudo chown -R $USER /cloudsql
-```
-
-You'll also need to initialize an environment variable containing the directory you just created:
-```bash
-export DB_SOCKET_DIR=/path/to/the/new/directory
+sudo mkdir ./cloudsql
+sudo chown -R $USER ./cloudsql
```

Comment:
I suspect `/cloudsql` was used so the same path could be used to test locally as would be used in the app engine environment (so that configuration files wouldn't have to be tweaked manually or have "if development" vs "if production" logic). Are you sure it works in both cases if you make this change?

And if so, you can drop the `sudo` and `chown` which were necessary only for manipulating a root-level directory.

Reply:
Done. Confirmed that this works running locally and on App Engine.